tests/migration/sle12_online_migration/snapper_rollback.pm

```diff
@@ -34,8 +34,7 @@ sub run {
     script_run "snapper rollback";
 
     # reboot into the system before online migration
-    script_run("systemctl reboot", 0);
-    reset_consoles;
+    power_action('reboot');
```

Comment:
Validation run now in PR description.

Reply:
It broke all of our online migration tests at snapper rollback step https://openqa.suse.de/tests/1954241#step/snapper_rollback/16.
From your verification run I guess you didn't do `real rollback` since it didn't load `grub_test_snapshot` step to choose before-migration snapshot in grub, i think it because `svirt` backend is powerless during grub2 stage. Probably no need snapper rollback step in case of `svirt` backend online migration without setting `ROLLBACK_AFTER_MIGRATION`.
And I will revert the change in online migration snapper rollback, we can't use `power_action` in a read-only snapshot with desktop, that is why we just use command directly.

Reply:
@mnowaksuse could you please take a look? Otherwise I suggest to revert.

Reply:
I am looking into it.

Reply:
Problem fixed by revert, rehash of the feature: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/5616. Sorry for the fuss @mitiao.

Reply:
@mnowaksuse never mind :)